integrationtests/tests/tests.rs

```diff
@@ -212,7 +212,7 @@ async fn drop_peer_3_during_epoch(fed: &FederationTest) {
 
     // let peers run consensus, but delay peer 0 so if peer 3 wasn't dropped peer 0 won't be included
     join_all(vec![
-        Either::Left(fed.subset_peers(&[1, 2]).run_consensus_epochs(1)),
+        Either::Left(fed.subset_peers(&[1, 2]).await_consensus_epoch()),
```

Comment:
Like this we don't check for empty proposals here..could this be a problem ?

Reply:
So it's okay to do this if the test writer wants to wait explicitly by calling `await_consensus_epoch` because they are doing something else concurrently that will trigger an epoch.